src/corelibs/U2View/src/ov_phyltree/TreeSettings.h

```diff
@@ -28,9 +28,12 @@
 #include <QVariant>
 
 namespace U2 {
-enum TREE_TYPE { DEFAULT,
-                 PHYLOGRAM,
-                 CLADOGRAM };
+
+enum TreeType {
```

Comment:
`enum class` to avoid problems with implicit conversion

Reply:
How to convert enum class from/to QVariant?

Reply:
Will do it at as separate PR if you provide details